### ISSUE TITLE
Floating point exception

### ISSUE THREAD

**Issue by bmario** [metricq/metricq-db-hta#10]
```
db-hta-dummy_1             | wait-for-it.sh: waiting 15 seconds for rabbitmq-server:5672
db-hta-dummy_1             | wait-for-it.sh: rabbitmq-server:5672 is available after 0 seconds
db-hta-dummy_1             | [2020-04-14 14:11:09.616216953 UTC][metricq][ INFO]: connecting to management server: amqp://admin:admin@rabbitmq-server/
db-hta-dummy_1             | [2020-04-14 14:11:09.618954457 UTC][metricq][DEBUG]: Using plaintext connection.
db-hta-dummy_1             | [2020-04-14 14:11:09.619819050 UTC][metricq][DEBUG]: resolved rabbitmq-server to 172.18.0.5:5672
db-hta-dummy_1             | [2020-04-14 14:11:09.620000301 UTC][metricq][DEBUG]: Established connection to rabbitmq-server at 172.18.0.5:5672
db-hta-dummy_1             | [2020-04-14 14:11:09.621432683 UTC][metricq][DEBUG]: Negotiated heartbeat interval to 60 seconds
db-hta-dummy_1             | [2020-04-14 14:11:09.623897131 UTC][metricq][DEBUG]: ConnectionHandler::onReady
db-hta-dummy_1             | [2020-04-14 14:11:09.624760474 UTC][metricq][DEBUG]: management channel ready
db-hta-dummy_1             | [2020-04-14 14:11:09.626858022 UTC][metricq][DEBUG]: management rpc sending db.register
db-hta-dummy_1             | [2020-04-14 14:11:09.642849730 UTC][metricq][DEBUG]: management rpc response received: {"dataServerAddress": "amqp://rabbitmq-server/", "dataQueue": "db-hta-dummy-data", "historyQueue": "db-hta-dummy-hreq", "config": {"_id": "db-hta-dummy", "_rev": "1-bfb7a9a6e23a23d5502f7a17b73a6c24", "threads": 4, "type": "file", "path": "/var/hta", "metrics": {"dummy.source": {"mode": "RW", "interval_min": 400000000, "interval_max": 400000000000000, "interval_factor": 10}}}}
db-hta-dummy_1             | [2020-04-14 14:11:09.642937054 UTC][metricq][DEBUG]: start parsing config
db-hta-dummy_1             | [2020-04-14 14:11:09.642974894 UTC][metricq][DEBUG]: start parsing data config
db-hta-dummy_1             | [2020-04-14 14:11:09.643006307 UTC][metricq][DEBUG]: opening data connection to amqp://admin:admin@rabbitmq-server/
db-hta-dummy_1             | [2020-04-14 14:11:09.643045468 UTC][metricq][DEBUG]: Using plaintext connection.
db-hta-dummy_1             | [2020-04-14 14:11:09.643098536 UTC][DEBUG]: on_db_config
db-hta-dummy_1             | [2020-04-14 14:11:09.643367413 UTC][DEBUG]: async directory setup
db-hta-dummy_1             | [2020-04-14 14:11:09.643501475 UTC][DEBUG]: async directory complete
db-hta-dummy_1             | [2020-04-14 14:11:09.643525557 UTC][metricq][DEBUG]: resolved rabbitmq-server to 172.18.0.5:5672
db-hta-dummy_1             | [2020-04-14 14:11:09.644866222 UTC][metricq][DEBUG]: Established connection to rabbitmq-server at 172.18.0.5:5672
db-hta-dummy_1             | [2020-04-14 14:11:09.654261685 UTC][metricq][DEBUG]: Negotiated heartbeat interval to 60 seconds
db-hta-dummy_1             | [2020-04-14 14:11:09.656999128 UTC][metricq][DEBUG]: ConnectionHandler::onReady
db-hta-dummy_1             | [2020-04-14 14:11:09.657806761 UTC][metricq][DEBUG]: data_channel ready
db-hta-dummy_1             | [2020-04-14 14:11:09.658228289 UTC][metricq][ INFO]: setting up data queue, messages 87205, consumers 0
db-hta-dummy_1             | [2020-04-14 14:11:09.658554997 UTC][metricq][ INFO]: setting up history queue, messages 4, consumers 0
db-hta-dummy_1             | [2020-04-14 14:11:09.723541466 UTC][metricq][DEBUG]: sink data queue consume success
db-hta-dummy_1             | Floating point exception (core dumped)
```

**Comment by bmario** [comment]
Was this fixed with 744264f785d5c0b870bb570149fb3c6edce67dd0 ?

**Comment by tilsche** [comment]
I think so
